### PR TITLE
Feature/roofer json format

### PR DESCRIPTION
**Pull request by IKatsuba**

### CODE REVIEW

File: apps/rxnode-article/src/app/app.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
+console.log(import('@angular/core'));
```
Comment (MarsiBarsi) [inline apps/rxnode-article/src/app/app.tsx]: лишнее?

File: libs/angular/builder/browser/schema.json [hunk 2]
```diff
@@ -4,6 +4,39 @@
   "description": "Browser target options",
   "type": "object",
   "properties": {
+    "roofer": {
+      "type": "object",
+      "description": "",
```
Comment (MarsiBarsi) [inline libs/angular/builder/browser/schema.json]: не добавить ли дескрипшн?

File: libs/angular/src/lib/services/loader.service.ts [hunk 3]
```diff
@@ -18,18 +18,22 @@ export class LoaderService {
...
-            combineLatest(
-              assets
-                .filter(file => file.endsWith('.js'))
-                .map(file => this.importJS(file)),
+      this.http.get<{groups: string[][]}>(assetMapUrl).pipe(
+        switchMap(({groups}) => {
+          return concat(
+            ...groups.map(assets =>
+              merge(
+                combineLatest(
```
Comment (MarsiBarsi) [inline libs/angular/src/lib/services/loader.service.ts]: чет не додумался, а что нам тут `combineLatest` дает?

Comment (IKatsuba) [reply libs/angular/src/lib/services/loader.service.ts]: рил лишний оператор

**Review by MarsiBarsi — APPROVED** [review]

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version webpack-plugin@0.1.0-alpha.4 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version angular@0.1.0-alpha.6 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version core@0.1.0-alpha.4 :tada:

The release is available on [GitHub release](<github_release_url>)

**Pull request merged**